parse_a_changelog.gemspec

```diff
@@ -1,21 +1,25 @@
-Gem::Specification.new do |s|
-  s.name = %q{parse_a_changelog}
-  s.version = `cat < VERSION`
-  s.authors = ["John Tuttle"]
-  s.email = "jtuttle.develops@gmail.com"
+Gem::Specification.new do |spec|
+  spec.name = "parse_a_changelog"
+  spec.version = `cat < VERSION`
+  spec.authors = ["John Tuttle"]
+  spec.email = "jtuttle.develops@gmail.com"
   
-  s.summary = %q{A validator for the keep-a-changelog format}
-  s.description = %q{Uses a grammar describing the keep-a-changelog format to attempt to parse a given file.}
-  s.homepage = "http://github.com/cyberark/parse-a-changelog"
-  s.license = "Apache-2.0"
+  spec.summary = %q{A validator for the keep-a-changelog format}
+  spec.description = %q{Uses a grammar describing the keep-a-changelog format to attempt to parse a given file.}
+  spec.homepage = "http://github.com/cyberark/parse-a-changelog"
+  spec.license = "Apache-2.0"
   
-  s.files = Dir.glob("{bin,lib}/**/*") + %w(LICENSE README.md CHANGELOG.md VERSION)
-  s.bindir = 'bin'
-  s.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
-  s.require_paths = ["lib"]
+  #spec.files = Dir.glob("{bin,lib}/**/*") + %w(LICENSE README.md CHANGELOG.md VERSION)
```

Comment:
not sure if this can just be removed or needs to stay commented

Reply:
Removed